### ISSUE TITLE
Update to Chromium v130

### ISSUE THREAD

**Issue by jdagdelen** [Igalia/wolvic-chromium#129]
Hi! Are there plans to pull in the upstream changes from chrome up to v130 soon? Is there anything we can do to help this process progress?

**Comment by svillar** [comment]
> Hi! Are there plans to pull in the upstream changes from chrome up to v130 soon? Is there anything we can do to help this process progress?

Absolutely. We do regularly update, but releases didn't leave much time for that. We'd love having help to carry on this work. We usually start from an upstream stable branch, and then we pile our changes on top. That usually involves updating APIs, implementing new classes, etc... and also checking that everything works as it should be and that there are no regressions.

Just contact us privately if you have plans to start/support that work.

**Comment by svillar** [comment]
We've already upgraded to M150. Closing this
